pkg/network/usm/sharedlibraries/ebpf.go

```diff
@@ -451,8 +461,6 @@ func (e *EbpfProgram) Stop() {
 
```

Comment:
Just above (Github won't let me put a comment on unchanged code), the comment states we are using atomics. We are not anymore so it needs updating.